langchain-plotly/README.md

```diff
@@ -0,0 +1,19 @@
+# Langchain Plotly Example
```

Comment:
maybe give more information on how to edit the prompt and explain that this is educational content that does not perform well.

Reply:
J'ai changé, tu en penses quoi?